### PR TITLE
Error order improvement

### PR DESCRIPTION
**Pull request by PhilippGackstatter**
Companion PR for iotaledger/iota.go#709.

### CODE REVIEW

File: pkg/network/protocols/core/protocol.go [hunk 1]
```diff
@@ -216,7 +216,7 @@ func (p *Protocol) onBlock(blockData []byte, id peer.ID) {
 
 func (p *Protocol) onBlockRequest(idBytes []byte, id peer.ID) {
 	if len(idBytes) != iotago.BlockIDLength {
-		p.Events.Error.Trigger(ierrors.Wrap(iotago.ErrInvalidIdentifierLength, "failed to deserialize block request"), id)
+		p.Events.Error.Trigger(ierrors.Errorf("failed to deserialize block request: invalid block id length"), id)
```
Comment (muXxer) [inline pkg/network/protocols/core/protocol.go]: Isn't there the `%s` missing?
Or do you want to use `ierrors.New` instead?

File: pkg/network/protocols/core/protocol.go [hunk 2]
```diff
@@ -237,7 +237,7 @@ func (p *Protocol) onSlotCommitment(commitmentBytes []byte, id peer.ID) {
 
 func (p *Protocol) onSlotCommitmentRequest(idBytes []byte, id peer.ID) {
 	if len(idBytes) != iotago.CommitmentIDLength {
-		p.Events.Error.Trigger(ierrors.Wrap(iotago.ErrInvalidIdentifierLength, "failed to deserialize slot commitment request"), id)
+		p.Events.Error.Trigger(ierrors.Errorf("failed to deserialize slot commitment request: invalid commitment id length"), id)
```
Comment (muXxer) [inline pkg/network/protocols/core/protocol.go]: Isn't there the `%s` missing?
Or do you want to use `ierrors.New` instead?

File: pkg/network/protocols/core/protocol.go [hunk 3]
```diff
@@ -294,7 +294,7 @@ func (p *Protocol) onAttestations(commitmentBytes []byte, attestationsBytes []by
 
 func (p *Protocol) onAttestationsRequest(commitmentIDBytes []byte, id peer.ID) {
 	if len(commitmentIDBytes) != iotago.CommitmentIDLength {
-		p.Events.Error.Trigger(ierrors.Wrap(iotago.ErrInvalidIdentifierLength, "failed to deserialize commitmentID in attestations request"), id)
+		p.Events.Error.Trigger(ierrors.Errorf("failed to deserialize commitmentID in attestations request: invalid commitment id length"), id)
```
Comment (muXxer) [inline pkg/network/protocols/core/protocol.go]: Isn't there the `%s` missing?
Or do you want to use `ierrors.New` instead?

**Review by muXxer — CHANGES_REQUESTED** [review]

File: pkg/network/protocols/core/protocol.go (hunk 1 above)
Comment (PhilippGackstatter) [reply pkg/network/protocols/core/protocol.go]: The `id` is passed to `Trigger` not to `Errorf`.

**Review by muXxer — APPROVED** [review]

**Pull request merged**